### ISSUE TITLE
Bug: Missing as-set when displaying as-path

### ISSUE THREAD

**Issue by sbng** [sbng/mrt2mmdb#4]
```
$ ./lookup.py --mmdb /tmp/out.mmdb --ipaddress 104.218.138.0                                                          [U!bgpscanner]
{
 "autonomous_system_number": 395480,
 "autonomous_system_organization": "ALKEON-CAPITAL-MANAGEMENT-LLC",
 "organization": "104.218.138.0/24",
 "isp": "42 2914 62833"
}

$bgpscanner data/mrt-dump.ams.202402171710.gz | grep 104.218.138.0
=|104.218.138.0/24|42 2914 62833 {13631,395480}|74.63.25.241|i||62833 104.218.141.254||74.63.25.241 42|1707660345|1

```
Missing {13631,395480}|

**Comment by sbng** [comment]
fixed in 7f91844

**Comment by sbng** [comment]
merge into main branch
